src/lib/Twig/Components/Chip.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\DesignSystemTwig\Twig\Components;
+
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+use Symfony\UX\TwigComponent\Attribute\PreMount;
+
+#[AsTwigComponent('ibexa:chip')]
+final class Chip
```

Comment:
```suggestion
final readonly class Chip
```

Reply:
I wouldn’t mark Chip as a final readonly class. Symfony UX Twig components rely on the runtime mutating public properties after instantiation (it hydrates them with the resolved props post-PreMount). Declaring the class readonly would block those assignments and trigger a ReadonlyPropertyError during component mounting. cc @albozek